packages/blockchain-wallet-v4-frontend/src/components/Brokerage/ConfirmAvailabilityRows/util.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
```

Comment:
Personally, I didn't expect to see components in this file. I suggest calling this `ParseDepositTerms.tsx` instead of `util.tsx`.

Reply:
yeah it didn't start out with components in it. I'll make that change.